__tests__/unit/fact.test.js

```diff
@@ -438,6 +441,102 @@ describe("getFact()", () => {
   });
 });
 
+describe("getFactTexts", () => {
+  test("returns correctly mutated object with types and facts", function () {
+    let obj = {};
+    getFactTexts(['math', 'trivia'], obj, 0, 10)
+
+    expect(obj['math'].length).toEqual(9);
+    expect(obj['trivia'].length).toEqual(11);
+  });
+})
+
+describe("getAllFacts(num)", () => {
+  test("return all types of facts for given number", function () {
+    let validNum = getAllFacts(24);
+
+    expect(validNum.number).toBe(24);
+    expect(validNum.year.length).toBeGreaterThan(1);
+    expect(validNum.date.length).toBeGreaterThan(1);
+    expect(validNum.math.length).toBeGreaterThan(1);
+    expect(validNum.trivia.length).toBeGreaterThan(1);
+  });
+
+  test("return all types with default messages for missing facts", function () {
+    let validNum = getAllFacts(1932);
+
+    expect(validNum.number).toBe(1932);
```

Comment:
This will be a problem if facts get added for 1932. We need to set up a mock db before these test is merged.

Reply:
Spoke with Nate, we're going to break this out into a separate ticket & go ahead with the merge